dace/dtypes.py

```diff
@@ -96,6 +96,7 @@ class Language(aenum.AutoNumberEnum):
 
     Python = ()
     CPP = ()
+    RTL = ()
```

Comment:
I think this should be `SystemVerilog` to be symmetric with the others. We don't support _any_ RTL language here!

Reply:
I agree, it is nicer for consistency of this enumeration, but this actually rises a bigger question on how general we want to keep or RTL CodeGen. Do we want to support other RTL languages in the future too through this CodeGen? (for example Verilog will also work for Verilator, and the Intel/Xiling tools).
I changed the language from RTL to SystemVerilog, but kept the target as rtl (i.e. target 'rtl', RTLCodeGen and the file is rtl.py)